### PR TITLE
Mapped lots of random things

### PR DESCRIPTION
**Pull request by Shnupbups**
Mostly function arguments, but plenty of other things too. Fixed some mistakes I found also.

### CODE REVIEW

File: mappings/net/minecraft/block/BlockState.mapping [hunk 1]
```diff
@@ -6,6 +6,7 @@ CLASS net/minecraft/class_2680 net/minecraft/block/BlockState
 		FIELD field_16558 opaque Z
 		FIELD field_16559 DIRECTIONS [Lnet/minecraft/class_2350;
 		FIELD field_16560 shapes [Lnet/minecraft/class_265;
+		FIELD field_19429 isSolidFullSquare [Z
```
Comment (asiekierka) [inline mappings/net/minecraft/block/BlockState.mapping]: solidFullSquare

File: mappings/net/minecraft/block/BlockState.mapping [hunk 2]
```diff
@@ -200,3 +201,4 @@ CLASS net/minecraft/class_2680 net/minecraft/block/BlockState
 		ARG 2 state
 		ARG 3 hitResult
 		ARG 4 projectile
+	METHOD method_20827 isSolidFullSquare (Lnet/minecraft/class_1922;Lnet/minecraft/class_2338;Lnet/minecraft/class_2350;)Z
```
Comment (asiekierka) [inline mappings/net/minecraft/block/BlockState.mapping]: How can a 3D model be a square? Or is this side-specific, in which case the method should probably say so?

File: mappings/net/minecraft/client/input/KeyCombo.mapping [hunk 3]
```diff
@@ -0,0 +1,7 @@
+CLASS net/minecraft/class_4324 net/minecraft/client/input/KeyCombo
```
Comment (asiekierka) [inline mappings/net/minecraft/client/input/KeyCombo.mapping:1]: KeyCombo? Not KeyCombination? It's also made out of chars...

File: mappings/net/minecraft/client/util/math/PosAndRot.mapping [hunk 4]
```diff
@@ -0,0 +1,11 @@
+CLASS net/minecraft/class_4462 net/minecraft/client/util/math/PosAndRot
```
Comment (asiekierka) [inline mappings/net/minecraft/client/util/math/PosAndRot.mapping:1]: That's a bad name

File: mappings/net/minecraft/realms/RealmsError.mapping [hunk 5]
```diff
@@ -0,0 +1,8 @@
+CLASS net/minecraft/class_4345 net/minecraft/realms/RealmsError
```
Comment (asiekierka) [inline mappings/net/minecraft/realms/RealmsError.mapping]: Are these not mapped by Rune's PR?

**Review by asiekierka — CHANGES_REQUESTED** [review]

**Comment by Shnupbups** [comment]
They may be mapped by Rune's PR, I didn't see it before making this one.

As for the names of `PosAndRot` and `KeyCombo`, they are the Mojang names. (as can be seen by their `toString` methods)

And as for `isSolidFullSquare` it is for checking whether a specific side of a block state is a full square. It's named the same as the method from `Block` that it calls.

File: mappings/net/minecraft/client/util/math/PosAndRot.mapping (hunk 4 above)
Comment (Shnupbups) [reply mappings/net/minecraft/client/util/math/PosAndRot.mapping:1]: It's a Mojang name. See the class' `toString`.

File: mappings/net/minecraft/client/input/KeyCombo.mapping (hunk 3 above)
Comment (Shnupbups) [reply mappings/net/minecraft/client/input/KeyCombo.mapping:1]: It's a Mojang name. See the class' `toString`.

File: mappings/net/minecraft/realms/RealmsError.mapping (hunk 5 above)
Comment (Shnupbups) [reply mappings/net/minecraft/realms/RealmsError.mapping]: Maybe. In all honesty I haven't checked.

File: mappings/net/minecraft/block/BlockState.mapping (hunk 2 above)
Comment (Shnupbups) [reply mappings/net/minecraft/block/BlockState.mapping]: It's for checking if a specific side of a blockstate is a solid full square. It uses the same name as the method it calls from `Block`.

File: mappings/net/minecraft/block/BlockState.mapping (hunk 1 above)
Comment (Shnupbups) [reply mappings/net/minecraft/block/BlockState.mapping]: Good point, will change.

File: mappings/net/minecraft/block/BlockState.mapping (hunk 2 above)
Comment (asiekierka) [reply mappings/net/minecraft/block/BlockState.mapping]: isSideSolidFullSquare, then

File: mappings/net/minecraft/realms/RealmsError.mapping (hunk 5 above)
Comment (asiekierka) [reply mappings/net/minecraft/realms/RealmsError.mapping]: Time to check!

Comment (Shnupbups) [reply mappings/net/minecraft/realms/RealmsError.mapping]: It is, as are a few other classes. I have removed them from this PR.

File: mappings/net/minecraft/block/BlockState.mapping (hunk 2 above)
Comment (Shnupbups) [reply mappings/net/minecraft/block/BlockState.mapping]: Well then should the method in `Block` not also be `isSideSolidFullSquare`?

Comment (asiekierka) [reply mappings/net/minecraft/block/BlockState.mapping]: Correct

Comment (Shnupbups) [reply mappings/net/minecraft/block/BlockState.mapping]: image
This is the code, the method in `Block` is also side specific and is named `isSolidFullSquare`. I named the `BlockState` method the same for consistency. I can change both, but I don't see why one should be different to the other.

Comment (Shnupbups) [reply mappings/net/minecraft/block/BlockState.mapping]: Done

**Comment by Shnupbups** [comment]
New version at #794

**Pull request closed without merging**